### PR TITLE
Added command to manually ban all alternate accounts

### PR DESCRIPTION
**Pull request by davidli3100**
- finds the discord ID's UWID
- finds all discord IDs associated with that UWID
- asynchronously bans all accounts if they are in the server

### CODE REVIEW

**Comment by davidli3100** [comment]
not sure what iterating through the entire guild's users using `.includes` would entail perf wise. May be better to just not care if a user exists and try/catch non-existent users

File: commands/banAlts.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+        message.channel.send(`Attempting to ban all alternate accounts of the user ${userID}`);
+        await message.guild.members.fetch();
+
+        // get specified user's UWID
+        const { uwid: userUWID } = await mongo.getDB().collection("users").findOne({ discordId: userID.id })
+        const altCounter = 0
+        // find all other discord accounts that share this UWID
+        for await (const user of mongo.getDB().collection("users").find({ uwid: userUWID })) {
+          // probably not very efficient for large servers, could just try/catch
```
Comment (sunny-zuo) [inline commands/banAlts.js:29]: From an efficiency standpoint this should be fine - going through the cache (locally) should have minimal performance impact and doesn't rely on API requests which has additional latency.

File: commands/banAlts.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          // errors if user doesn't exist instead
+          if (message.guild.members.cache.values().includes(user.discordId)) {
```
Comment (sunny-zuo) [inline commands/banAlts.js:31]: Is this the right syntax? I believe the cache is a collection of <Snowflake, GuildMember>, and calling `.values()` would get `GuildMember`s. IIRC you can use `.has()` to fetch the guild member assuming the Snowflake (discordId) exists

**Comment by davidli3100** [comment]
Checked typedefs and `.has` should work

**Pull request merged**